events/views.py

```diff
@@ -13,6 +13,27 @@ def index(request):
     return render(request, "events/events.html", {"events": events})
 
 
+@login_required
+def updateEvent(request, event_id):
+    event = get_object_or_404(Event, pk=event_id)
+    if request.method == "POST":
+        # Update the event with data from the form
+        event_location_id = request.POST["event_location_id"]
+        location_object = Location.objects.get(id=event_location_id)
+        event.event_location = location_object
+        event.event_name = request.POST.get("event_name")
+        event.start_time = request.POST.get("start_time")
+        event.end_time = request.POST.get("end_time")
+        event.capacity = request.POST.get("capacity")
+        # Save the updated event
+        event.save()
+
+        return redirect("events:index")  # Redirect to the event list or a success page
+
```

Comment:
When I test locally, I noticed that if you update inputs other than the location, the "update" button will direct to an error page (even though the database seems to get updated). It looks that the location id is having problem here.
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/73798f9e-8a03-41ac-b6ae-c61c31d2afca)